ddtrace/contrib/pytest/plugin.py

```diff
@@ -102,6 +104,10 @@ def pytest_runtest_protocol(item, nextitem):
         span.set_tag(test.SUITE, item.module.__name__)
         span.set_tag(test.TYPE, SpanTypes.TEST.value)
 
+        if getattr(item, "callspec", None):
```

Comment:
should we add a comment on that `callspec` is only available if it is a parametrized test?